.gitbook/FAQ.md

```diff
@@ -143,7 +142,7 @@ Technically, an L1 token can have multiple representations on L2. This mean that
 
 [And here's the relayer service Boba uses for relaying messages from L2 to L1](https://github.com/bobanetwork/boba\_legacy/tree/develop/packages/message-relayer).
 
-<figure><img src="./assets/testing and testnet.png" alt=""><figcaption></figcaption></figure>
+## Testing and Testnet
```

Comment:
I'm a big fan of replacing headings as images with proper headings -- I'm really unsure why we did this in the first place.